src/sentry/grouping/strategies/message.py

```diff
@@ -103,6 +103,9 @@
         ='([^']+)' |
         ="([^"]+)"
     ) |
+    (?P<quoted_identifier>
+        "([a-z0-9_])+"
```

Comment:
dashes?

Reply:
single quotes?

Reply:
<img width="403" alt="image" src="https://github.com/getsentry/sentry/assets/44410/5f31ac89-2433-4c2d-b2bf-0715e217cc1d">
<img width="507" alt="image" src="https://github.com/getsentry/sentry/assets/44410/2023512a-b3b8-4f13-884c-e12ef7ecc74a">

Reply:
Possibly not what we want since it would increase the chances of unintended consequences

Reply:
I think a positive lookahead for a digit will do what you originally wanted here: `(["'])(?=.*\d)\1[^"']*?\1`